web/html/src/manager/content-management/project/project.renderer.js

```diff
@@ -5,9 +5,17 @@ import Project from './project';
 window.pageRenderers = window.pageRenderers || {};
 window.pageRenderers.contentManagement = window.pageRenderers.contentManagement || {};
 window.pageRenderers.contentManagement.project = window.pageRenderers.contentManagement.project || {};
-window.pageRenderers.contentManagement.project.renderer = (id) => {
+window.pageRenderers.contentManagement.project.renderer = (id, {project, wasFreshlyCreatedMessage} = {}) => {
+  let projectJson = {};
+  try{
+    projectJson = JSON.parse(project);
+  }  catch(error) {}
+
   ReactDOM.render(
-    <Project />,
+    <Project
+      project={projectJson}
+      { ...( wasFreshlyCreatedMessage && { wasFreshlyCreatedMessage } ) }
```

Comment:
ahah kind of :P
 Anyway I will simplify it with https://github.com/lneves12/uyuni/blob/56c202c179feecb8229f7ac72a06eb5506f4a029/web/html/src/manager/content-management/list-projects/list-projects.renderer.js#L18 and change the name to "flashMessage" too.

That's basically passing optional parameters through destructuring to <Project />.
If wasFreshlyCreatedMessage is false it will not pass any prop and if it's defined it will pass wasFreshlyCreatedMessage={wasFreshlyCreatedMessage}.